### ISSUE TITLE
Facing error "This version of ChromeDriver only supports Chrome version 111"

### ISSUE THREAD

**Issue by staz9** [wkeeling/selenium-wire#660]
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:63241
from session not created: This version of ChromeDriver only supports Chrome version 111
Current browser version is 110.0.5481.177

I installed the latest chromium and still getting this error. Anyway to fix it?

**Comment by PatrickVdWillik** [comment]
This error is unrelated to Selenium-wire aand is most likedly caused by your selenium driver referencing the incorrect binary. 
 
Try restarting your application, if that doesn't work, try restarting the terminal from which you are running your application. I was presented with this error in the last few days too and restarting my app/terminal resolved the issue. Alternatively, if you installed the lastest chromium in a custom location, use the Options to specify where.

**Comment by calebTree** [comment]
Yes, not related to selenium-wire. One must choose the correct browser driver or not specify one at all i.e. `driver = webdriver.Chrome()` which should pick up the first available driver.

**Comment by staz9** [comment]
Solved
